test/e2e/disk_encryption.go

```diff
@@ -20,7 +20,7 @@ var _ = Describe("Encryption at host should be enabled", func() {
 		ctx := context.Background()
 
 		By("getting the resource group where the VM instances live in")
-		oc, err := clients.OpenshiftClustersv20200430.Get(ctx, vnetResourceGroup, clusterName)
+		oc, err := clients.OpenshiftClustersv20220904.Get(ctx, vnetResourceGroup, clusterName)
```

Comment:
I'll be adding additional checks to these two tests: want to check that cluster resource has appropriate fields set